lib/reducers/create-otp-reducer.js

```diff
@@ -520,6 +521,9 @@ function createOtpReducer (config, initialQuery) {
           ? update(state, { user: { recentSearches: { $splice: [[index, 1]] } } })
           : state
       }
+      case 'SET_CURRENT_USER': {
```

Comment:
Since we're setting a precedent of having separate files for actions, I think we should have separate files for reducers as well.